chapters/Chapter 1 — Untitled.md

```diff
@@ -61,23 +64,19 @@ Leanne was not far behind: her gravity generator tried to keep her on the floor;
 
 Later, she’d tell herself she intended it this way all along. She’d put the suit on, after all.
 
-Had she not been wearing the generator, Leanne assumed she’d be spinning fast. Instead, she was floating calmly through the apparent emptiness, breathing fresh oxygen from the tanks on her back.
-
-Her ship was in a desperate spin. The engines had stopped: the ship was just smart enough to detect it was spinning, but not quite smart enough to stop the spin. She’d figure out how to stop it later.
-
-The other ship was twice as large as hers. It looked patched together from pieces of other ships, but Leanne was rather certain it had been designed that way: it had been the style in the 2870’s, and was still popular amongst some. Its seven side engines seemed rather weak; Leanne supposed most of the power was reserved for the rear engines. She looked away as they flashed bright green. 
+Had she not been wearing the generator, Leanne assumed she’d be spinning fast. Instead, she was floating calmly through the apparent emptiness, breathing fresh oxygen from the tanks on her back. Her ship was in a desperate spin. The engines had stopped: the ship was just smart enough to detect it was spinning, but not quite smart enough to stop the spin. She’d figure it out later.
 
-Leanne guessed their Alts were helping them to dexterously take the ship out of its spin, now that they were trying to get away. 
+The other ship was twice as large as hers. It looked patched together from pieces of other ships, but Leanne was rather certain it had been designed that way: it had been the style of the 2870’s, and Leanne guessed some still found it popular. Its seven side engines seemed rather weak; Leanne supposed most of the power was reserved for the rear engines.
 
-She glanced to her left. She wasn’t sure if they’d come back for their partner. But she had a rule. She had to.
+Now that they were only trying to get away, Leanne assumed, their Alt must have been trying to help them. Their ship’s engines were firing in careful, controlled bursts, dexterously taking it out of its spin. Leanne looked away, her head spinning, as the bright green flashes assaulted her eyes.
 
-She raised her gun. The man who had boarded her ship wasn’t that far away. They had been blown out more or less together. 
+She glanced to her left. The man who had boarded her ship wasn't that far away. She'd been blown out more or less with him. She wasn’t sure if his partner would come back for him. He was probably as good as dead; if he didn't run out of air, some debris would puncture his suit: while maneuverable, it was too light to offer any real protection. But Leanne had a rule.
```

Comment:
"But Leanne had a rule." Ick. I'm not sure if it is actually bad, but I don't like it.